rero/rero-ils-ui
# serial: receive an issue

Creates components and services allowing to receive an issue item.

* Creates component for 'serial' type holding.
* Adapts `ItemsRoute` to pre-populate the item editor with holding data
  in case where user add/update an issue item.
* Adapts and corrects `HoldingRoute` allowing to display an holding
  detail view.
* Moves `item.service` from CirculationModule to AppModule.
* Adapts the document detail view to correctly display item from serial
  type holding.

Co-authored-by : Renaud Michotte <renaud.michotte@gmail.com>

## Dependencies

Need to use https://github.com/rero/rero-ils/tree/US1275-receive-issue rero-ils branch

## How to test?

- Search about a serial document. Once found, display the detail view of this document.
- in the "Get" tab, you could see the already received issue. In the corresponding holding header, a new icon allows to receive issues for this holding : 
![image](https://user-images.githubusercontent.com/10031585/83016524-601bfa80-a022-11ea-8696-607eafbf2264.png)
- in the "receive an issue" page, try all functionalities : 
  * quick receive an expected issue
  * edit and receive an expected issue
  * add an irregular issue
  * delete a received issue
  * edit an received issue

![image](https://user-images.githubusercontent.com/10031585/83016834-e46e7d80-a022-11ea-9cd7-f411a981f785.png)

## Code review check list
